kairos-l1-utils/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "kairos-l1-query"
+version.workspace = true
+edition.workspace = true
+license.workspace = true
+
+[dependencies]
+casper-client = {git="https://github.com/casper-ecosystem/casper-client-rs", branch="release-2.0.0"}
```

Comment:
Why are we using release 2.0.0

Reply:
Because that is the latest stable release and generally Casper Client is pretty bad. The dev branch breaks when you try to import it as a library.

Reply:
I'll remove it and default the id to 0

Reply:
2.0.0 is available on crates.io, what revision do you need exactly and why?